### DISCUSSION TITLE
v2/zh/config/plugins/others

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#314] [Announcements]
# v2/zh/config/plugins/others

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/config/plugins/others.html

**Comment by veeking** [comment]


**Reply by Mister-Hope** [reply]
是的，有什么问题么

**Reply by Mister-Hope** [reply]
阅读时间显示是可以关掉的，麻烦仔细看文档。其他的自己详见组件替换

**Reply by Mister-Hope** [reply]
能不能™好好看文档。

- https://vuepress-theme-hope.github.io/v2/zh/guide/feature/page-info.html
- https://vuepress-theme-hope.github.io/v2/zh/config/theme/layout.html#pageinfo
- https://vuepress-theme-hope.github.io/v2/zh/config/frontmatter/layout.html#pageinfo

我哪句话告诉你 readingTime 是可以配置成 false的？

- https://vuepress-theme-hope.github.io/v2/zh/config/plugins/others.html#readingtime
<img width="627" alt="image" src="https://user-images.githubusercontent.com/33315834/193187169-1345f5c8-cc66-484b-8002-fa9ed1bbe0f2.png">

可以禁用的我哪个没说？
<img width="642" alt="image" src="https://user-images.githubusercontent.com/33315834/193187224-45e247a5-cccf-4ad7-bfc5-ea9c861053b0.png">
